tests/PhpSpreadsheetTests/Functional/DrawingImageHyperlinkTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: yuzhakov
+ * Date: 08.05.18
+ * Time: 12:00.
+ */
```

Comment:
We do not use file comment blocks, please remove. You will be credited in the commit information.